src/dndbeyond/base/utils.js

```diff
@@ -196,7 +196,7 @@ async function buildAttackRoll(character, attack_source, name, description, prop
             await queryDamageTypeFromArray(roll_properties.name, damages, damage_types, ["Radiant", "Necrotic"]);
         }
         
-        if (to_hit) {
+        if (to_hit || force_damages_only) {
```

Comment:
I think you can do a full roll without `force-damages-only`. Like clicking the beyond20 button on the sneak attack action would count as a normal roll, without "force_damages_only" being set to true.

Reply:
![image](https://user-images.githubusercontent.com/7988297/151906635-fb44e77c-ca88-4dcc-b265-6a6a445d57d8.png)

This somehow sets `force_damages_only`, and has no to-hit, so it won't run damagesToCrits without this, then fails like this:
![image](https://user-images.githubusercontent.com/7988297/151906698-816d9b23-e728-4b7e-9e3c-99b25f60f709.png)

![image](https://user-images.githubusercontent.com/7988297/151906790-c8aa2953-f064-4b53-a026-c128796cbf14.png)

Reply:
Yeah but clicking here with force crit : 
![image](https://user-images.githubusercontent.com/27990/151907618-fb3af9f3-3557-4587-8421-5886efee5fe3.png)
or on that button : 
![image](https://user-images.githubusercontent.com/27990/151907648-299761cf-33a9-403f-ba79-004004300768.png)

Will not cause `force_damages_only` to be set, so you'd get the same error because `damagesToCrits` won't be able to find the crits.. would need to have it check if to_hit or force crits... which isn't the available info at that time.

Reply:
Well crap...